script/test_in_openshift/lib.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+function convert::print_msg () {
+    echo ""
+    tput setaf 4
+    tput bold
+    echo -e "$@"
+    tput sgr0
+    echo ""
+}
+
+function install_oc_client () {
+    sudo sed -i 's:DOCKER_OPTS=":DOCKER_OPTS="--insecure-registry 172.30.0.0/16 :g' /etc/default/docker
+    sudo mv /bin/findmnt /bin/findmnt.backup
+    sudo cat /etc/default/docker
+    sudo /etc/init.d/docker restart
+    # FIXME
+    wget https://github.com/openshift/origin/releases/download/v1.4.1/openshift-origin-client-tools-v1.4.1-3f9807a-linux-64bit.tar.gz -O /tmp/oc.tar.gz 2> /dev/null > /dev/null
+    mkdir /tmp/ocdir && cd /tmp/ocdir && tar -xvvf /tmp/oc.tar.gz > /dev/null
+    sudo mv /tmp/ocdir/*/oc /usr/bin/
+}
```

Comment:
It does make sense, but people may want to run this test on their host system. I know I'd like to, instead of having to rely on Travis each time. @kadel ?